modules/olm-creating-fb-index-image.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-custom-catalogs.adoc
+
+ifdef::openshift-origin[]
+:registry-image: quay.io/openshift/origin-operator-registry:4.9.0
+endif::[]
+ifndef::openshift-origin[]
+:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v4.9
+endif::[]
+
+[id="olm-creating-fb-index-image_{context}"]
+= Creating a file-based index image
+
+You can create an index image that uses a plain text _file-based catalog_ format (JSON or YAML) instead of the deprecated SQLite database format. The `opm` CLI has new tooling that helps initialize an index in the file-based format, render new records into it, and then validate that the index is valid.
+
+.Prerequisites
+
+* `opm` version 1.18+
```

Comment:
Should we reference a downstream version here?

I think I misled you when we chatted. When I said `1.18+` I meant "some number of commits after 1.18.0" was tagged.

If we stick with an upstream version, I think this should be `1.19+`, which we'll be releasing in the next week or so.

Reply:
I think we should use the downstream version since those docs are OCP releases docs, not OKD.